cli/setup.py

```diff
@@ -51,6 +51,12 @@ def readme():
     # This does not impact any functionality. Also next connexion release will lift this limitation
     # and we will be able to upgrade.
     "connexion==2.7.0",
+    # pin to connexion transitive dependencies
+    "clickclick>=1.2,<21",
+    "jsonschema>=2.5.1,<4",
+    "requests>=2.9.1,<3",
+    "inflection>=0.3.1,<0.6",
+    "openapi-spec-validator>=0.2.4,<0.4",
```

Comment:
Why giving a range instead of `~=`?
Does dependabot work with the range?

Reply:
I left them exactly as they were defined in the connexion source linked to in the PR description. Theoretically we could use `~=` to avoid using a range, but the goal was to use the same constraints as connexion.